### PR TITLE
Updated the behaviour of evictionHard

### PR DESCRIPTION
**Pull request by vaibhav2107**
Ref : #36706 

This PR added the undocumented behaviour of eviction Hard to the docs.

### CODE REVIEW

**Comment by vaibhav2107** [comment]
CC @sftim

**Comment by vaibhav2107** [comment]
Hi @daminisatya ,@mtaufen
Could you please have a look

File: content/en/docs/concepts/scheduling-eviction/node-pressure-eviction.md [hunk 1]
```diff
@@ -154,6 +154,12 @@ The kubelet has the following default hard eviction thresholds:
 * `imagefs.available<15%`
 * `nodefs.inodesFree<5%` (Linux nodes)
 
+These default values of hard eviction thresholds will only be set if none 
+of the parameter is changed. If you changed the value of any of the parameter,
```
Comment (chrisnegus) [inline content/en/docs/concepts/scheduling-eviction/node-pressure-eviction.md:158]:
```suggestion
of the parameters is changed. If you changed the value of any of the parameter, 
```

**Review by chrisnegus — COMMENTED** [review]
One suggestion.

File: content/en/docs/concepts/scheduling-eviction/node-pressure-eviction.md (hunk 1 above)
Comment (divya-mohan0209) [inline content/en/docs/concepts/scheduling-eviction/node-pressure-eviction.md:158]: Combining the suggestion by Chris above along with a slight amendment in grammar.
```suggestion
of the parameters is changed. If you changed the value of any parameter, 
```

**Review by divya-mohan0209 — DISMISSED** [review]
Slight nit.

**Comment by vaibhav2107** [comment]
@chrisnegus @divya-mohan0209 I have applied the suggested changes.
PTAL.

File: content/en/docs/tasks/administer-cluster/kubelet-config-file.md [hunk 2]
```diff
@@ -42,6 +42,11 @@ and evict Pods when available memory drops below 200Mi.
 All other Kubelet configuration values are left at their built-in defaults, unless overridden
 by flags. Command line flags which target the same value as a config file will override that value.
 
+Note: In the example, by changing the default value of only one parameter for
+evictionHard, the default values of other parameters will not be inherited and
+will be set to zero. In order to provide custom values, you should provide all
+the threshold values respectively.
```
Comment (tengqm) [inline content/en/docs/tasks/administer-cluster/kubelet-config-file.md:48]:
```suggestion
{{< note >}}
In the example, by changing the default value of only one parameter for
evictionHard, the default values of other parameters will not be inherited and
will be set to zero. In order to provide custom values, you should provide all
the threshold values respectively.
{{< /note >}}
```
```

**Comment by vaibhav2107** [comment]
@tengqm PTAL

File: content/en/docs/concepts/scheduling-eviction/node-pressure-eviction.md [hunk 3]
```diff
@@ -154,6 +154,12 @@ The kubelet has the following default hard eviction thresholds:
 * `imagefs.available<15%`
 * `nodefs.inodesFree<5%` (Linux nodes)
 
+These default values of hard eviction thresholds will only be set if none 
+of the parameters is changed. If you changed the value of any parameter, 
+then the values of other parameters will not be inherited as the default 
+values and will be set to zero. In order to provide custom values, you 
+should provide all the thresholds respectively.
```
Comment (sftim) [inline content/en/docs/concepts/scheduling-eviction/node-pressure-eviction.md:161]: Personally I'd pop this in a [note](https://kubernetes.io/docs/contribute/style/style-guide/#note) callout here as well, but that's not a critical change to make.

File: content/en/docs/tasks/administer-cluster/kubelet-config-file.md [hunk 4]
```diff
@@ -42,6 +42,13 @@ and evict Pods when available memory drops below 200Mi.
 All other Kubelet configuration values are left at their built-in defaults, unless overridden
 by flags. Command line flags which target the same value as a config file will override that value.
 
+{{< note >}}
+In the example, by changing the default value of only one parameter for
+evictionHard, the default values of other parameters will not be inherited and
```
Comment (sftim) [inline content/en/docs/tasks/administer-cluster/kubelet-config-file.md:47]: (nit)
```suggestion
`evictionHard`, the default values of other parameters will not be inherited and
```

**Review by sftim — COMMENTED** [review]
Thanks!

/approve

The feedback here is for hints, and doesn't mean it needs to be actioned.

**Pull request merged**